docs/examples/005-viz-with-holoviz.ipynb

```diff
@@ -0,0 +1,431 @@
+{
```

Comment:
<p>Rephrase: "If you wish to plot <em>only</em> the grid's geometrical elements such as nodes, edges (i.e. without any data variables mapped to them), you can use the <code>Grid.to_geodataframe()</code> method to obtain a <code>GeoDataFrame</code> with a singular geometry column having each face represented as a <code>MultiPolygon"</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UXARRAY/uxarray/pull/428/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>